i18n/en/code.json

```diff
@@ -286,7 +286,7 @@
     "message": "A constraint-based record & functional language mainly used in configuration and policy scenarios."
   },
   "homepage.feature.konfig.description": {
-    "message": "Kusion Model is a pre-built, KCL-described configuration model in KusionStack that provides users with an out-of-the-box, highly abstracted configuration interface."
+    "message": "Konfig is a monorepo that stores operation intentions described by KCL and provides an out-of-the-box cloud-native application model, allowing users to quickly start the journey of cloud-native delivery."
```

Comment:
an out-of-the-box cloud-native application model
---
a set of out-of-the-box cloud-native application models？

Reply:
fixed